dockyard/api/app.py

```diff
@@ -0,0 +1,42 @@
+import pecan
+
+from dockyard.api import config as api_config
+from oslo_config import cfg
+from pecan import make_app
+
+from dockyard import model
+
+
+API_SERVICE_OPT = [
+    cfg.PortOpt('port',
+                 default=5869,
+                 help='Port for the dockyard service.'),
+    cfg.IPOpt('host',
+              default='127.0.0.1',
```

Comment:
This address should be 0.0.0.0 so that anyone from remote machine is able to communicate to this server.